Sources/Sails/Application.swift

```diff
@@ -63,8 +65,16 @@ public class Application {
 }
 
 extension Application: Verifier {
-    public func verify(_ method: Method, _ uri: String) -> Int {
-        requestsMap[RequestMade(method: method, uri: uri), default: 0]
+    public func verify(_ method: Method, _ uri: String, times: UInt? = 1, file: StaticString = #filePath, line: UInt = #line) {
```

Comment:
default value for `times` needs to be `nil` 😬

otherwise we'll never get past that first conditional